demo/Signs.swift

```diff
@@ -1354,6 +1362,14 @@ extension Sign {
                 return nil
             case .speedLimitEndZone:
                 return nil
+            case .informationRestrictedParking:
```

Comment:
nit: Should we have the same order as for

https://github.com/mapbox/vision-sdk-trainer/blob/2826e98e245d8d42c2658874ed9e47b26fe4d4a3/trainers/classifier/export/export_utils.py#L214

for convenience?

Reply:
nevermind :)

Reply:
Hm, I can see the same order.